.npmignore

```diff
@@ -18,23 +18,17 @@ lighthouse-logger/
 # keep the chrome launcher script
 !lighthouse-core/scripts/manual-chrome-launcher.js
 
-# excluding cli/test minus smokehouse
-lighthouse-cli/test
-!lighthouse-cli/test/smokehouse
-!lighthouse-cli/test/fixtures/
-
-!lighthouse-cli/index.js
-lighthouse-cli/*.map
-lighthouse-cli/commands/*.map
-lighthouse-cli/types/*.js
-lighthouse-cli/types/*.map
+# keep smokehouse tests, etc
+!lighthouse-cli/test
```

Comment:
how much bigger does our package end up?

Reply:
oh this was not fun. turns out `yarn pack` builds things VERY differently than `npm pack`. 
and `npm pack` gives basically the same package before and after this PR. wonderful.

Reply:
turns out it's smaller now. because the npmignore in master includes all of lighthouse-cli already.

so aint that somethin